htdocs/core/class/canvas.class.php

```diff
@@ -145,7 +145,7 @@ function displayCanvasExists($action)
     {
         if (empty($this->template_dir)) return 0;
 
-        if (file_exists($this->template_dir.($this->card?$this->card.'_':'').$this->_cleanaction($action).'.tpl.php')) return 1;
+        if (file_exists($this->template_dir.((!empty($this->card) && $this->card != 'list')?$this->card.'_':'').$this->_cleanaction($action).'.tpl.php')) return 1;
```

Comment:
The canvas is a field into the table of object. So it is a property of an object.
So using canvas for list has no sense (a list can contains several record). 
What means $this->card = 'list' ?

The problem here is in product/list.php where there is some code to try to use "canvas" where we should not try in such a page.
If you need a dedicated list page, why not using the hook printFieldListXXX or creating your own php page ?

Reply:
@eldy 
yes I put this correction so that it takes "list.tpl.php" because otherwise it looked for "list_list.tpl.php"!
but precisely I want to delete the call of a template in the list of products, and modify to have fields specific to a canvas in the new list engine